controller/main_controller.php

```diff
@@ -71,10 +76,44 @@ public function handle_who_is_online()
 		));
 
 		// Output page
-		page_header('');
+		//page_header('');
 		$this->template->set_filenames(array(
 			'body' => 'ajax_base/statistics.html')
 		);
-		page_footer();
+		//page_footer();
+
+		$this->template->display('body');
+
+		garbage_collection();
+		exit_handler();
+	}
+
+	/**
+	* Handle the Private msgs and notification requests
+	*
+	* @return null
+	* @access public
+	*/
+	public function handle_pm_note()
+	{
+		$mode = request_var('mode', '');
+
+		// Output page
+		page_header('');
+
+		$this->template->assign_vars(array(
+			'S_AJAX_BASE_NOTIFICATION'	=> ($mode == 'notification') ? true : false,
+			'S_AJAX_BASE_PMREGBAR'		=> ($mode == 'pmregbar') ? true : false,
+			'U_VIEW_ALL_NOTIFICATIONS'	=> generate_board_url() . "/ucp.php?i=ucp_notifications",
```

Comment:
Please, use `append_sid` to ensure that `session_id` is perserved when stored in URL only.